src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,31 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			// nonce check if search form submitted
+			$nonce_check = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? wp_verify_nonce( $_POST['aggregator']['nonce'], 'aggregator_' . $this->tab->get_slug() . '_request' ) : 1;
+			if( $nonce_check !== false ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: d5ce5e866e0600b8aed5032f2a614de4</sub>